core/modules/widgets/edit-text.js

```diff
@@ -230,12 +243,11 @@ EditTextWidget.prototype.fixHeight = function() {
 		var newHeight = Math.max(domNode.scrollHeight + domNode.offsetHeight - domNode.clientHeight,minHeight);
 		// Only try to change the height if it has changed
 		if(newHeight !== domNode.offsetHeight) {
-			domNode.style.height =  newHeight + "px";
+			domNode.style.height = newHeight + "px";
 			// Make sure that the dimensions of the textarea are recalculated
 			$tw.utils.forceLayout(domNode);
-			// Check that the scroll position is still visible before trying to scroll back to it
-			scrollTop = Math.min(scrollTop,self.document.body.scrollHeight - window.innerHeight);
```

Comment:
Hi @Jermolene 

This is intentional because I use scrollTop and not `window.scrollTo()`

> scrollTop can be set to any integer value, with certain caveats:
> 
> If the element can't be scrolled (e.g. it has no overflow or if the element is non-scrollable), scrollTop is set to 0.
> If set to a value less than 0, scrollTop is set to 0.
> If set to a value greater than the maximum that the content can be scrolled, scrollTop is set to the maximum.

> https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollTop

So I do not need to do this check.

Moreover, the new code that accounts for situation with nested elements that act as scroll containers, the parent may not be the window object (as in the original/old code; also reflected in $tw.utils.getScrollPosition only looking at the window object) and the scrollHeight of relevance may not be the one of the body element.